charter.md

```diff
@@ -22,14 +22,20 @@ Server-initiated services are out of scope. The working group will first deliver
 a requirements document for IP proxying. Once both are complete, the working group will focus on a protocol solution for
```

Comment:
Since we're rechartering, I would remove the text about our past deliverables - they're no longer our goal/purpose

Reply:
Good point!

Reply:
Thought about that initially and was trying to keep it a minimal diff, but I'll cut these lines out since we did do them and just leave the rest of the "focus on initial set" since that is a true statement about the group's focus (i.e. server-initiated services remain out of scope).

Reply:
Removed in [8af81ca](https://github.com/ietf-wg-masque/wg-materials/pull/7/commits/8af81ca9873dc5f769a87707af732f24be6455bf)